source/as-promise/calculate-retry-delay.ts

```diff
@@ -6,13 +6,15 @@ import {
 	RetryFunction
 } from './types';
 
+type Returns<T extends (...args: any) => any, V> = (...args: Parameters<T>) => V;
```

Comment:
```suggestion
type Returns<T extends (...args: any) => unknown, V> = (...args: Parameters<T>) => V;
```

Can this be `unknown`?